src/native.js

```diff
@@ -80,12 +183,18 @@ export const hasNonNativeBidder = adUnit =>
  * @return {Boolean} If object is valid
  */
 export function nativeBidIsValid(bid, {index = auctionManager.index} = {}) {
+  const bidRequest = index.getAdUnit(bid);
+  if (!bidRequest) { return false; }
+
+  if (deepAccess(bid, 'native.ortb') && deepAccess(bidRequest, 'nativeParams.ortb')) {
```

Comment:
This seems to enforce that an ortb response is valid only if the request was also in ortb. But during the transition period I imagine we'll have to deal with a lot of "legacy" requests, which would force adapters to individually have logic like "if the request is legacy, respond in legacy; otherwise respond in ortb". Is it possible to decouple them? maybe by falling back to the conversion logic if the request does not have `nativeParams.ortb`.

If I understand this right with this PR no adapter is hitting this code path. Could/should the PBS adapter respond in ortb?

Also a minor point, I don't like the name `bidRequest` for an ad unit.

Reply:
1. if `ortb` keyword is not present in `nativeParams`, prebid will assume that the request was made using the legacy format. What do you mean by "decouple" the two request types?  Do you mean to convert the request to OpenRTB and then validate the response against the just-created openRTB request? 

2. With this PR we will adapt all adapters to translate ortb to legacy format request. PBS will understand both types, checkout prebidServerBidAdapter/index.js . 

3. I will change bidRequest to adUnit.

Reply:
1 - yes, because as adapters transition, they will ideally want to deal only with ORTB - so for a legacy request they will convert to ortb and reply with ortb. This check would now be skipped if the request is legacy.
2. But the PBS adapter is still replying with legacy responses. Should it be converted over to ortb completely? (should it even deal with legacy requests, beside converting them?)

Reply:
1. All responses are converted to Ortb. 
2. Prebid Server was already responding in Ortb. I only had to remove the code that converted to legacy.